etp-front/src/pages/valvonta-kaytto/toimenpiteet.js

```diff
@@ -669,3 +669,102 @@ export const determineProcessPhaseFromToimenpiteet = toimenpiteet => {
     return 'decision-order';
   }
 };
+
+export const primaryTransitionForToimenpidetype = (
+  toimenpiteet,
+  toimenpidetypes
+) => {
+  let primaryToimenpidetype;
+
+  const lastCreatedToimenpide = R.last(toimenpiteet);
+  let toimenpideTobaseTheAllowedToimenpidetypesOn = lastCreatedToimenpide;
```

Comment:
```suggestion
  let toimenpideToBaseTheAllowedToimenpideTypesOn = lastCreatedToimenpide;
```
Ehkä vielä ennemmin jotain muuta mutta en osaa sanoa mitä